cmd/create/operatorroles/cmd.go

```diff
@@ -207,6 +207,16 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	managedPolicies, err := r.AWSClient.HasManagedPolicies(cluster)
+	if err != nil {
+		r.Reporter.Errorf("Failed to determine if cluster has managed policies: %v", err)
+		os.Exit(1)
+	}
+	if managedPolicies && env == ocm.Production {
```

Comment:
Please add a comment that this is temporary until AWS managed policies are introduced

Reply:
Added.